src/elements/emby-checkbox/emby-checkbox.js

```diff
@@ -6,6 +7,11 @@ import 'webcomponents.js/webcomponents-lite';
 const EmbyCheckboxPrototype = Object.create(HTMLInputElement.prototype);
 
 function onKeyDown(e) {
+    // Ignore GamepadA onKeyDown if using gamepadtokey since it will click the checkbox onKeyUp.
+    if (appSettings.enableGamepad && e.key === 'GamepadA') {
+        return;
+    }
```

Comment:
`enableGamepad` is a function, but you are just checking if it exists (always `true`).